ExportScripts/export_standalone_monolithic.py

```diff
@@ -222,14 +227,29 @@ def process_command(args: list,
 
     #commands are based on 
     #https://github.com/o3de/o3de-multiplayersample/blob/development/Documentation/PackedAssetBuilds.md
-
+    selected_platform = args.platform
+    if not selected_platform:
+        logger.info("Platform not specified! Defaulting to Host platform...")
+        system_platform = platform.system()
+        if not system_platform:
+            logger.error("Unable to identify host platform! Please supply the platform using '--platform'. Options are [pc, linux].")
+            sys.exit(1)
+        if system_platform == "Windows":
```

Comment:
How about a string to lower capitalization conversion before comparing the strings?

Reply:
Why not make the arguments a choice box from the possible values of platform.system()?